cloud_utils/cache/stores/redis.py

```diff
@@ -24,7 +24,7 @@ async def wrapper(*args, **kwargs):
 def make_store(
     make_redis_client: Callable[[], redis.Redis],
     max_parallelism: int,
-    ttl: int,
+    ttl: Callable[[Any], int],
```

Comment:
This is a breaking change and this lib is used in many of our repos.
You will have to go over the repos that use it and update the calls.

Reply:
Alternatively consider creating a new function that receives the ttl as a function and keep the original.